react-js/src/subviews/feature-list.tsx

```diff
@@ -1,15 +1,64 @@
 import React from 'react';
 import FeatureListItem from "./feature-list-item";
 import {Features} from "../model/features";
+// @ts-ignore
+import { ImagePicker, FilePicker } from 'react-file-picker';
+import {RoutePath} from "../service/routing-service";
 
 export default class FeatureList extends React.Component<any, any> {
 
     render() {
         return (
             <div>
-                {Features.LIST.map(feature =>
-                    <FeatureListItem key={feature.id} onClick={() => {this.props.onItemClick(feature)}} data={feature}/>
-                )}
+                {Features.LIST.map(feature => {
```

Comment:
this looks pretty terrible, but i think I implemented this hack myself because picker needed a hotfix so it would work on all devices and platforms. i'll refactor it myself